libs/akita/src/lib/dispatchers.ts

```diff
@@ -18,6 +19,6 @@ export function dispatchAdded(storeName: string) {
 }
 
 // @internal
-export function dispatchUpdate(storeName: string) {
-  $$updateStore.next(storeName);
+export function dispatchUpdate(storeName: string, action: StoreSnapshotAction) {
```

Comment:
Thanks for your PR.

In those days, I didn't have time to write tests for the devtools feature. Do you think this is something you can add?

You can mock the dev-tools and test that they get the correct actions.

Reply:
Added a test suite for devtools with some basic log action tests.

Reply:
Thanks!

We'll go out with v5 next week. If you think about more things that can be improved and require breaking changes, let me know.